RayLin.md

```diff
@@ -1,12 +1,43 @@
-# 07/18 ~ 07/22
+# 07/25 ~ 07/29
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - UI discussion with Peko
-  - Bug 681553 - [meta] Make HTML5 \<audio\> controls less of a hack
-    - No clear goals for now. Need define what we want controls optimized for audio, and UX should do that defining.
-    - De-prioritized. Save this bug to after visual refresh.
+    - Prefer design 01
+    - Consider special cases:
```

Comment:
I think first patch will be uploaded late this week as Peko might get confirmation this Wed.